src/actions/vstudio/vs2010_nuget.lua

```diff
@@ -0,0 +1,88 @@
+--
+-- vs2010_nuget.lua
+-- Generate a NuGet packages.config file.
+-- Copyright (c) 2016 Jason Perkins and the Premake project
+--
+
+	premake.vstudio.nuget2010 = {}
+
+	local p = premake
+	local vstudio = p.vstudio
+	local nuget2010 = p.vstudio.nuget2010
+
+
+--
+-- These functions take the package string as an argument and give you
+-- information about it.
+--
+
+	function nuget2010.packageName(package)
+		return package:gsub(":", ".")
+	end
+
+	function nuget2010.packageId(package)
+		return package:sub(0, package:find(":") - 1)
+	end
+
+	function nuget2010.packageVersion(package)
+		return package:sub(package:find(":") + 1, -1)
+	end
+
+	local function packageProject(wks, package)
+		for prj in p.workspace.eachproject(wks) do
+			if prj.nuget then
+				for i = 1, #prj.nuget do
+					local projectPackage = prj.nuget[i]
+
+					if projectPackage == package then
+						return prj
+					end
+				end
+			end
+		end
+	end
+
+	function nuget2010.packageFramework(wks, package)
+		local prj = packageProject(wks, package)
+
+		if prj.language == "C++" then
```

Comment:
Use `project.iscpp()` and `project.isdotnet()` here instead of string literal compares.